aten/src/ATen/native/mkldnn/MKLDNNConversions.cpp

```diff
@@ -86,14 +86,44 @@ Tensor mkldnn_reorder_conv2d_weight(
   return new_with_itensor_mkldnn(std::move(result), self.options());
 }
 
+Tensor mkldnn_reorder_conv3d_weight(
+    const Tensor& self,
+    IntArrayRef padding,
+    IntArrayRef stride,
+    IntArrayRef dilation,
+    int64_t groups) {
+
+  auto stride_vec = expand_param_if_needed(stride, "stride", 3);
```

Comment:
out of curiosity, why do you need to expand stride etc here, but don't need to do it for 2d conv? If it were not for these expansion calls, reorder_weight functions are exactly the same for 2d and 3d.

Reply:
Yes, we don't need expand them, they have been expanded at https://github.com/pytorch/pytorch/blob/13bd5992d0f91141322a2a70260648f4d333e87e/torch/nn/modules/conv.py#L402-L405